src/pages/home-page/home-page.module.scss

```diff
@@ -70,20 +70,93 @@
     font: $small-paragraph-font;
 }
 .cardsLayout {
-    display: flex;
+    display: grid;
     flex-direction: row;
     padding-left: 8vw;
     padding-right: 8vw;
     column-gap: 20px;
+    grid-template-rows: 1fr;
+    grid-template-columns: 1fr 1fr 1fr 1fr;
 }
 .HPprgrp {
     font: $large-paragraph;
     margin-bottom: 40px;
 }
 .hero-paragraph {
     text-align: center;
+    width: 100vw;
 }
 .hero-title {
     margin-top: 80px;
     font: $extra-large-thin-title;
 }
+
+.productCard {
+    width: 20vw;
+}
+
+// --- MEDIA QUERIES ---
+
+@media (1400px <=width) {
+    .root {
+        background-color: aqua;
+    }
+
+    .content {
+        max-width: 1400px;
+    }
+}
+
+@media (980px <=width < 1400px) {
+    .root {
+        background-color: red;
+    }
+}
+
+@media (420px <=width < 980px) {
+    .root {
+        background-color: plum;
+    }
+
+    .cardsLayout {
```

Comment:
why so much duplicated?

Reply:
removed unneeded duplicates, some are there to play with so I kept them for now